homeassistant/components/nuimo.py

```diff
@@ -0,0 +1,108 @@
+"""
+components.nuimo
+~~~~~~~~~~~~~~~~~~
+Component that connects to a Bluetooth reachable Nuimo device.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/nuimo/
+"""
+from logging import getLogger
+import threading
+from homeassistant.const import EVENT_HOMEASSISTANT_STOP
+from homeassistant.helpers import validate_config
+
+REQUIREMENTS = [
+    'https://github.com/shaftoe/nuimo-raspberrypi-demo/archive/'
+    'pip.zip#nuimo_core==0.0.1',
+    'https://github.com/IanHarvey/bluepy/archive/master.zip#bluepy==1.0.3']
```

Comment:
If this is a dependency that is required for `nuimo_core`, it shuld be part of that setup.py so that it will be automatically installed when nuimo_core get's installed